### PR TITLE
fixed styling of author link on category page

### LINKED ISSUE #144: Link to author page on category cards
Now that we are including author pages in this project, need to link the author name from the Category cards.

- For when the card images are the background behind the text, they are the standard color blue links, which means they will likely fail accessibility for 'contrast'. Sling team approves of making this a compliant color/style.
- So we should keep the link color as white when on the medium + large cards, but they might need to be underlined. They can stay blue when the text appears under the images as they are today.

Example pages:
https://www.sling.com/whatson/movies
https://www.sling.com/whatson/entertainment/home-and-food

image

### PR DESCRIPTION
**Pull request by rusmeenkhan1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #144 

Test URLs:
- Before: https://main--sling--aemsites.aem.page/whatson/movies
- After: https://144-link-to-authorpage--sling--aemsites.aem.page/whatson/movies

home & food
Test URLs:
- Before: https://main--sling--aemsites.aem.page/whatson/entertainment/home-and-food
- After: https://144-link-to-authorpage--sling--aemsites.aem.page/whatson/entertainment/home-and-food

### CODE REVIEW

**Comment by rusmeenkhan1** [comment]
Hi Team, i saw previously it was showing some contrast warnign with the older link-color. So i have updated it to a nearby matching color to avoid contrast warning. 
Screenshot 2024-07-16 at 2 48 25 PM

I have chosed below color:

File: aemedge/blocks/category/category.css [hunk 1]
```diff
@@ -119,6 +119,13 @@ main .section.category-container {
 
 .category .card-large .card-link .card-content .card-author-date .card-author {
   color: white;
+  text-decoration: underline;
```
Comment (helms-charity) [inline aemedge/blocks/category/category.css:122]: Only underline if the name is linked (currently it is not).

**Review by helms-charity — CHANGES_REQUESTED** [review]
The styling looks good, but should only look like links when the names are linked.

**Review by helms-charity — APPROVED** [review]
Looks good

**Pull request merged**